proprietary/tokens/src/components/ams/button.tokens.json

```diff
@@ -56,6 +56,10 @@
           "background-color": { "value": "transparent" },
           "color": { "value": "{ams.color.neutral-grey2}" }
         }
+      },
+      "icon-only": {
```

Comment:
I missed this before: the token should have ‘position’ in it as well.

I would split it already: `"icon-position": { "only": …`